style/client.css

```diff
@@ -208,6 +208,13 @@ button:disabled {
 	box-shadow: none;
 	color: #F9F9F9;
 }
+.dark .button.disabled,
+.dark .button.disabled:hover,
+.dark .button.disabled:active {
+	background: #555555;
+	border-color: #555555;
+	box-shadow: none;
```

Comment:
Even if you're not going to add a new `color`, you should re-set colors whenever you set background colors (and vice versa) for readability reasons.